.github/workflows/coverage.yml

```
@@ -0,0 +1,70 @@
+name: Coverage
+
+on:
+  push:
+    - master
```

I think you need to specify the branches.
```suggestion
    branches:
      - master
```